server/routes/v1/user.js

```diff
@@ -9,5 +10,6 @@ const router = express.Router();
 
 router.post('/signup', validateUserSignUp, existingUserCheck, signup);
 router.post('/login', validateUserLogin, authentication, login);
+router.get('/verify', verifyAccount);
```

Comment:
Does this mean this route is accessible from anywhere no protection just come right through? You can have some to check there is a token present before I hit the controller. It should give a 403 status code

Reply:
Ok.